### PR TITLE
11.0 mig project_timesheet_time_control

### PR DESCRIPTION
**Pull request by ernestotejeda**
* Update README
* Adapt tests
* Redefine the new kanban and form view of account.analytic.line
	* Redefine the new form view to add a 'Close/Open task' button and the 'Stop' button
	* Redefine the new kanban view to add the 'Stop' button

Cc @Tecnativa

### CODE REVIEW

File: project_timesheet_time_control/models/account_analytic_line.py [hunk 1]
```diff
@@ -75,3 +74,10 @@ def button_close_task(self):
                       "mark any.")
                 )
             line.task_id.write({'stage_id': stage.id})
+
+    @api.one
```
Comment (pedrobaeza) [inline project_timesheet_time_control/models/account_analytic_line.py]: @api.one is deprecated

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
You have an outdated branch, so runbot is not installing the module due to a missing dependency. Please fetch latest remotes and rebase the branch over them.

**Comment by pedrobaeza** [comment]
@ernestotejeda I think a good improvement for this module is to design a kanban view for the timesheet page with big button for stopping current time. Do you know what I'm talking about?

**Comment by ernestotejeda** [comment]
In the Kanban view I have put a small button with link style next to the place where the time is shown. Do you refer to the same button but bigger?

**Comment by pedrobaeza** [comment]
Yes, please try with the web_responsive + mobile view in the browser to check that is feasible to tap.

**Comment by ernestotejeda** [comment]
@pedrobaeza, I have changed the look of 'Stop' button on kanban view. Could you review again please?

**Comment by pedrobaeza** [comment]
@ernestotejeda I still don't see the button on the mobile view:
seleccion_027
so I don't know which view you are modifying.

There's also an extra problem when using a mobile. If you click on create new line, you get a totally unusable form view with fields that are not needed for timesheets, and other needed that are missing. It even don't get the passed defaults and I'm not able to write on the description field (but that's maybe a problem on Firefox only). Can you please fix this?
seleccion_028

**Comment by ernestotejeda** [comment]
Done.

It's true, I was working on another view, I thought you were referring to the kanban view of account.analytic.line shown in the Timesheets menu -> Timesheet -> My Timesheets (or All Timesheets). When you access that menu, it shows a kanban view that was not there before, that's where I added the 'Stop' button.

File: project_timesheet_time_control/readme/USAGE.rst [hunk 2]
```diff
@@ -0,0 +1,24 @@
+You can access via timesheets:
+
+#. Go to Timesheets > My Timesheet > Detailed Activities.
```
Comment (cubells) [inline project_timesheet_time_control/readme/USAGE.rst]: Old path is now `Timesheets > Timesheet > All Timesheets`

File: project_timesheet_time_control/readme/USAGE.rst [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+#. Create a new record.
+#. You will see now that the "Date" field contains also time information.
+#. If you don't select any "project", you will be able to select any "task",
+   opened or not.
+#. Selecting a "task", the corresponding "project" is filled.
+#. Selecting a "project", tasks are filtered for only allow
+   to select opened tasks for that project. Remember that an opened task is
+   a task whose stage doesn't have "Closed" mark checked.
+#. At the end of the line, you will see an icon of a cross inside a circle.
```
Comment (cubells) [inline project_timesheet_time_control/readme/USAGE.rst]: `you will see an stop button`

**Review by cubells — APPROVED** [review]
Minor changes

LGTM

Code review and test.

**Comment by pedrobaeza** [comment]
@ernestotejeda there's a problem when you add a line from the new form view:

seleccion_043

You will probably need to pass a default value + invisible fields. Please check.

**Review by chienandalu — APPROVED** [review]
Tested on runbot :+1:

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**